src/Pages/Home.js

```diff
@@ -78,7 +78,7 @@ export default class Home extends Component{
         ];
 
         return (
-            <Table responsive>
+            <Table responsive className="home-table">
```

Comment:
There is too much padding between icons and stats
![image](https://github.com/user-attachments/assets/7a9368b9-b9cf-4ba7-a754-bf4b5db858ad)